examples/src/pages/Migration.tsx

```diff
@@ -388,16 +389,15 @@ export function Migration() {
               {
                 const { pointIndex, point } = intersection;
                 console.log(`Clicked point with pointIndex ${pointIndex} at`, point);
-                const sphere = new THREE.Mesh(new THREE.SphereBufferGeometry(0.1), new THREE.MeshBasicMaterial({ color: 'red' }));
```

Comment:
Why we want to remove it?

Reply:
On clicking in point cloud for measurement tool will add sphere, so this is removed to avoid multiple sphere at same position